vmaas/utils/rpm.go

```diff
@@ -141,3 +141,14 @@ func (n *Nevra) Cmp(other *Nevra) int {
 	}
 	return n.EVRACmp(other)
 }
+
+func JoinNevra(name string, evr Evr, arch string) string {
+	if name == "" || evr.Release == "" || evr.Version == "" || arch == "" {
+		return ""
+	}
+	var epoch string
+	if evr.Epoch > 0 {
+		epoch = fmt.Sprintf("%d:", evr.Epoch)
+	}
+	return fmt.Sprintf("%s-%s%s-%s.%s", name, epoch, evr.Version, evr.Release, arch)
+}
```

Comment:
can be removed as this functionality is handled by `func (n *Nevra) String() string`. If we want to return empty string in some cases then we should do it also in the String() or StringE() functions. Also it might not be correct to return empty string if evr.Release is missing, there are some packages without Release version if I'm not mistaken. Let's handle these cases separately and fix it in existing functions in rpm.go file